src/components/Pagination/Pagination.js

```diff
@@ -0,0 +1,74 @@
+import { useMemo } from 'react';
+import Link from 'next/link';
+
+import config from '../../../package.json';
+import { Helmet } from 'react-helmet';
+
+import { GrPrevious as PreviousIcon, GrNext as NextIcon } from 'react-icons/gr';
+import styles from './Pagination.module.scss';
+
+const MAX_NUM_LINKS = 10;
+
+const { homepage = '' } = config;
+
+const Pagination = ({ pagesCount, currentPage, basePath }) => {
+  const path = `${basePath}/page/`;
+
+  const hasPreviousPage = pagesCount > 1 && currentPage > 1;
+  const hasNextPage = pagesCount > 1 && currentPage < pagesCount;
+
+  const pages = useMemo(() => {
+    let pages = [];
+    for (let i = 1; i < pagesCount + 1; i++) {
+      pages.push(i);
+    }
+    return pages;
+  }, [pagesCount]);
+
+  return (
+    <>
+      <Helmet>
+        {!hasPreviousPage && <link rel="canonical" href={`${homepage}${basePath}`} />}
+        {hasPreviousPage && <link rel="prev" href={`${homepage}${path}${currentPage - 1}`} />}
+        {hasNextPage && <link rel="next" href={`${homepage}${path}${currentPage + 1}`} />}
+      </Helmet>
+
+      <nav className={styles.nav}>
+        {hasPreviousPage && (
+          <Link href={`${path}${currentPage - 1}`}>
+            <a className={styles.prev} aria-label="Previous Posts">
+              <PreviousIcon /> Previous
+            </a>
+          </Link>
+        )}
+
+        <div className={styles.pages}>
```

Comment:
can we move this into an unordered list? `<ul>`

Reply:
sure thing! making the change

Reply:
oh interesting 🤔  i would suspect `<ul>` would be the right way to go, right?

Reply:
https://www.a11ymatters.com/pattern/pagination/

Reply:
Yes the `ul` fits perfectly in my opinion. Thanks for the link, I will add the additional things like the role